src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableLink.tsx

```diff
@@ -1,23 +1,58 @@
 import React, { useContext } from 'react';
-import Link, { LinkProps } from '@admin/components/Link';
-import wrapEditableComponent, {
-  EditingContext,
-} from '@common/modules/find-statistics/util/wrapEditableComponent';
+import classNames from 'classnames';
+import { LinkProps } from '@admin/components/Link';
+import { EditingContext } from '@common/modules/find-statistics/util/wrapEditableComponent';
 import Button from '@common/components/Button';
 
 interface Props extends LinkProps {
+  to: string;
   removeOnClick: () => void;
 }
 
-const EditableLink = ({ removeOnClick, ...props }: Props) => {
+const EditableLink = ({
+  removeOnClick,
+  to: href,
+  children,
+  unvisited,
+  className,
+  ...props
+}: Props) => {
   const { isEditing } = useContext(EditingContext);
 
   return !isEditing ? (
-    <Link {...props} />
+    <a
+      className={classNames(
+        'govuk-link',
+        {
+          'govuk-link--no-visited-state': unvisited,
+        },
+        className,
+      )}
+      href={href}
+      {...props}
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      {children}
+    </a>
   ) : (
     <div>
       <div>
-        <Link {...props} />
+        <a
```

Comment:
You may wanna reuse the "Link" Component here?

Reply:
The Link component is of `'react-router-dom'`
https://reacttraining.com/react-router/web/api/Link

AFAIK doesn't support linking like an anchor with a href, only within the app routes

I've adapted `@admin`'s Link to act differently in the case where a `href` is supplied and `to == false`